opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -152,3 +154,57 @@ def _python_path_without_directory(python_path, directory, path_separator):
         "",
         python_path,
     )
+
+
+_OTEL_SEMCONV_STABILITY_OPT_IN_KEY = "OTEL_SEMCONV_STABILITY_OPT_IN"
```

Comment:
Is this env var named used prior across languages? If not, we should probably add *_PYTHON_*

Reply:
Good question. I will bring this up with the TC.

Reply:
Shown in [migration plan](https://github.com/open-telemetry/semantic-conventions/tree/main/docs/http#semantic-conventions-for-http)